### PR TITLE
Fix: use clevis in addition to systemd-cryptenroll

### PR DESCRIPTION
**Pull request by grydz**

### CODE REVIEW

File: ansible/packer_tdx_playbook.yml
[the remark this answers is not in the document]
Comment (grydz) [reply ansible/packer_tdx_playbook.yml:1]: Whoops, this is a fail when I rebased.

File: resources/scripts/cosmian_fstool [hunk 1]
```diff
@@ -135,7 +135,20 @@ fi
...
+
+# Remove previous entry from the fstab and crypttab (create a .bak file to ease the rollback)
+echo "Removing previous obsolete auto mounting rules..."
+sed -i.bak "/$CONTAINER_MAPPING_NAME/d" /etc/crypttab
+sed -i.bak "/$CONTAINER_MAPPING_NAME/d" /etc/fstab
+
+# # Auto decrypt & auto mount the luks when rebooting (if not already done)
+echo "Enabling auto mounting when rebooting..."
+grep -qF -- "$CONTAINER_MAPPING_NAME $CONTAINER_PATH" /etc/crypttab || echo "$CONTAINER_MAPPING_NAME $CONTAINER_PATH - tpm2-device=auto,headless=true" >>/etc/crypttab
+grep -qF -- "$CONTAINER_MAPPING_PATH $CONTAINER_MOUNT_PATH" /etc/crypttab || echo "$CONTAINER_MAPPING_PATH $CONTAINER_MOUNT_PATH ext4 defaults 0 0" >>/etc/fstab
```
Comment (Manuthor) [inline resources/scripts/cosmian_fstool:149]: this block is repeated twice

Comment (grydz) [reply resources/scripts/cosmian_fstool]: Thanks, fixed in ab4e8fcb2e38978db797c40bb0376b37da202baf.

**Comment by grydz** [comment]
Closing in favor of https://github.com/Cosmian/cosmian_vm/pull/98.

**Pull request closed without merging**